src/app/tests/suites/certification/Test_TC_MF_1_3.yaml

```diff
@@ -16,391 +16,153 @@ name: 22.1.3. [TC-MF-1.3] Node Behavior using ECM [DUT - Commissionee]
-          [1635864972.825467][6022:6027] CHIP:DMG: {
-          [1635864972.825558][6022:6027] CHIP:DMG: 0x0 = 2,
-          [1635864972.825613][6022:6027] CHIP:DMG: 0x1 = [
-          [1635864972.825693][6022:6027] CHIP:DMG: 0x4, 0x5b, 0x93, 0x38, 0xab,
-          0x50, 0x6, 0x85, 0xc0, 0x50, 0xfe, 0x7d, 0x6c, 0xeb, 0x9d, 0xbd, 0x55,
-          0x90, 0x5e, 0x4f, 0x54, 0xed, 0x94, 0x21, 0xc0, 0x8d, 0x9d, 0x4f,
-          0x60, 0x48, 0xdb, 0x38, 0x4a, 0xf4, 0xe2, 0xa3, 0xf9, 0xd0, 0x55,
-          0xd5, 0x63, 0xd0, [1635864972.825760][6022:6027] CHIP:DMG: ]
-          [1635864972.825821][6022:6027] CHIP:DMG: 0x2 = 45896,
-          [1635864972.825882][6022:6027] CHIP:DMG: 0x3 = 0,
-          [1635864972.825942][6022:6027] CHIP:DMG: 0x4 = 2,
-          [1635864972.826002][6022:6027] CHIP:DMG: 0x5 = ',
-          [1635864972.826095][6022:6027] CHIP:DMG: },
-          [1635864972.826172][6022:6027] CHIP:DMG: ],
-          [1635864972.826246][6022:6027] CHIP:DMG: DataElementVersion = 0x0,
-          [1635864972.826292][6022:6027] CHIP:DMG: },
-          [1635864972.826360][6022:6027] CHIP:DMG:
-          [1635864972.826406][6022:6027] CHIP:DMG: ],
-          [1635864972.826471][6022:6027] CHIP:DMG:
-          [1635864972.826520][6022:6027] CHIP:DMG: }
-          [1635864972.826691][6022:6027] CHIP:ZCL: ReadAttributesResponse:
-          [1635864972.826732][6022:6027] CHIP:ZCL: ClusterId: 0x0000_003E
-          [1635864972.826776][6022:6027] CHIP:ZCL: attributeId: 0x0000_0001
-          [1635864972.826816][6022:6027] CHIP:ZCL: status: Success (0x0000)
-          [1635864972.826858][6022:6027] CHIP:ZCL: attribute TLV Type: 0x16
-          [1635864972.826923][6022:6027] CHIP:TOO:
-          OnOperationalCredentialsFabricsListListAttributeResponse: 2 entries
-          [1635864972.827030][6022:6027] CHIP:TOO: FabricDescriptor[1]:
-          [1635864972.827067][6022:6027] CHIP:TOO: fabricIndex: 1
-          [1635864972.827105][6022:6027] CHIP:ZCL: RootPublicKey: 65
-          [1635864972.827143][6022:6027] CHIP:TOO: vendorId: 30024
-          [1635864972.827182][6022:6027] CHIP:TOO: fabricId: 0
-          [1635864972.827215][6022:6027] CHIP:TOO: nodeId: 1
-          [1635864972.827251][6022:6027] CHIP:ZCL: Label:
-          [1635864972.827298][6022:6027] CHIP:TOO: FabricDescriptor[2]:
-          [1635864972.827338][6022:6027] CHIP:TOO: fabricIndex: 2
-          [1635864972.827375][6022:6027] CHIP:ZCL: RootPublicKey: 65
-          [1635864972.827440][6022:6027] CHIP:TOO: vendorId: 45896
-          [1635864972.827479][6022:6027] CHIP:TOO: fabricId: 0
-          [1635864972.827517][6022:6027] CHIP:TOO: nodeId: 2"
-      disabled: true
+    #Issue 16094 Need to check length of list and verify its content
+    - label: "Query fabrics list"
+      identity: "beta"
+      command: "readAttribute"
+      cluster: "Operational Credentials"
+      attribute: "Fabrics"
+      response:
+          value: [{ Label: "", nodeId: nodeId2 }]
```

Comment:
We should also test that doing a non-fabric-filtered write returns both fabrics, right?

Reply:
Actually, the test step does indicate to validate both. Thank you. I apologies for all these inconsistencies. I uploaded the YAML from GRL without reviewing it. Updating.

Reply:
Added.